src/bit-systems/scene-loading.ts

```diff
@@ -34,9 +47,10 @@ function* loadScene(world: HubsWorld, eid: number, signal: AbortSignal, environm
       return;
     }
 
+    add(world, scene, loaderEid);
     // TODO: Use a unique id for each scene as the root nid
```

Comment:
This TODO is now slightly more appropriate above where we use "scene" as the nid for `setInitialNetworkedData`